packages/server/src/organization/organization.controller.ts

```diff
@@ -889,19 +893,21 @@ export class OrganizationController {
       });
     }
 
-    const fileName =
-      organization.id +
-      '-' +
-      Math.random().toString(36).substring(2, 15) +
-      Math.random().toString(36).substring(2, 15);
+    const fileName = organization.id + '-' + Date.now().toString() + '.webp';
+
+    // Create the upload location if it doesn't exist
+    if (!fs.existsSync(process.env.UPLOAD_LOCATION)) {
+      fs.mkdirSync(process.env.UPLOAD_LOCATION, { recursive: true });
+    }
 
-    const image = await Jimp.read(file.buffer); // Load the image
-    image.resize(256, Jimp.AUTO); // Resize the image to 256 pixels (width, maintaining aspect ratio)
-    await image.writeAsync(
-      path.join(process.env.UPLOAD_LOCATION as string, fileName),
-    ); //same old
+    const targetPath = path.join(process.env.UPLOAD_LOCATION, fileName);
 
-    organization.logoUrl = fileName;
+    try {
+      await sharp(file.buffer).resize(256).webp().toFile(targetPath);
```

Comment:
I realise that previously we also resized to 256x256, but I think we can resize it to 48x48 since that's the size of it in the image elements

Reply:
I have reduced this to 100x100 because 48x48 was still looking too obviously pixelated on my 1080p monitor.